test/support/test_complex.h

```diff
@@ -85,16 +85,16 @@ run_test()
 //         IF_DOUBLE_SUPPORT(test<int>())
 //         // ...
 //     }
-#define IF_DOUBLE_SUPPORT(x)                                                                          \
-    TestUtils::invoke_test_if(HasDoubleSupportInRuntime(), [](){ x; });
-#define IF_DOUBLE_SUPPORT_L(x)                                                                        \
-    TestUtils::invoke_test_if(HasDoubleSupportInRuntime(), x);
+#define IF_DOUBLE_SUPPORT(...)                                                                        \
```

Comment:
Changes in this file please see in https://github.com/oneapi-src/oneDPL/pull/667
In this PR all these changes only to avoid other compile errors.